ensembl/datacheck/app.py

```diff
@@ -263,12 +263,12 @@ def job_result(job_id):
     elif job['output']['failed_total'] > 0:
         job['status'] = 'failed'
     elif job['output']['passed_total'] == 0:
-        job['status'] = 'failed'
+        job['status'] = 'dc-failures'
```

Comment:
```suggestion
        job['status'] = 'error'
```
I suggest changing the name of the status, to avoid potentially confusing terminology and clearly distinguish between datachecks failing (which is valid, albeit not the desired outcome) and generating actual errors.

Reply:
@james-monkeyshines Not sure that `error` is less confusing, :-)  but what about `dc-run-error`?

Reply:
Sure, I agree that's better then 'error'.